test/testTargetsApp/Manifest.json

```diff
@@ -16,6 +16,12 @@
     "resource": "source/resource",
     "translation": "source/translation"
   },
+  "externalResources": {
+    "script": [ 
+        "https://localhost:2000/IDSCloudApp/IDSCloudApp:ext-js/socketcluster/socketcluster-client.min.js",
+        "http://localhost/compiled/source/cv/cv:libs/EventRecorder.js" 
```

Comment:
What's the deal with the colon again? I have just looked into the docs but couldn't find anything on that - was that some sort of micro-protocol? If yes, we should document it, otherwise it looks like a bug.

Reply:
no - i misunderstood the error, I though that was the actual URL, ie a directory name with a colon in it.